tests/libyui/tmux-uitest.sh

```diff
@@ -0,0 +1,106 @@
+#!/bin/bash
+# tmux-uitest.sh - a shell library to test Text User Interface using tmux
+
+SESSION=uitest
+: "${VERBOSE=false}"
+
+# $1 shell command for sh -c
+tmux_new_session() {
+    if $VERBOSE; then
+        echo Starting session
+    fi
+    # -s session name
+    # -x width -y height,
+    # -d detached
+    # FIXME: sleep to be able to see errors when running $1
+    tmux new-session -s "$SESSION" -x 80 -y 24 -d sh -c "$1; sleep 9999"
```

Comment:
@lslezak you asked "How can I get more details about the failures?"

This sleep lets us capture the screen after a command fails and exits.

Tmux has a `remain-on-exit` option but I haven't figured out how to enable it yet :-(

Reply:
Ha! Totally intuitive for a beginner... NOT

`tmux set-hook -g session-created 'set remain-on-exit on' \; new-session sh -c ...`

Found in https://github.com/tmux/tmux/issues/787